mobile/lib/services/local/local_sync_util.dart

```diff
@@ -34,17 +34,26 @@ Future<Tuple2<List<LocalPathAsset>, List<EnteFile>>> getLocalPathAssetsAndFiles(
     if (assetsInPath.isEmpty) {
       result = const Tuple2({}, []);
     } else {
-      result = await Computer.shared().compute(
-        _getLocalIDsAndFilesFromAssets,
-        param: <String, dynamic>{
-          "pathEntity": pathEntity,
-          "fromTime": fromTime,
-          "alreadySeenLocalIDs": alreadySeenLocalIDs,
-          "assetList": assetsInPath,
-        },
-        taskName:
-            "getLocalPathAssetsAndFiles-${pathEntity.name}-count-${assetsInPath.length}",
-      );
+      try {
+        result = await Computer.shared().compute(
+          _getLocalIDsAndFilesFromAssets,
+          param: <String, dynamic>{
+            "pathEntity": pathEntity,
+            "fromTime": fromTime,
+            "alreadySeenLocalIDs": alreadySeenLocalIDs,
+            "assetList": assetsInPath,
+          },
+          taskName:
+              "getLocalPathAssetsAndFiles-${pathEntity.name}-count-${assetsInPath.length}",
+        );
+      } catch (e) {
+        _logger.severe("_getLocalIDsAndFilesFromAssets failed", e);
+        _logger.info(
+          "Failed for pathEntity: ${pathEntity.name}",
+        );
+        result = const Tuple2({}, []);
```

Comment:
Won't this result in nothing getting imported? Let's discuss about this in person before merging this case.
May we can test the baheviour by throwing exception when entity name is equal to one of the local file on the device..